resources/control-runtime/postinst

```diff
@@ -87,6 +87,16 @@ restartIfFlagFileExists(){
   return 0
 }
 
+checkJava(){
+  # Java must be at least version 8, check for this
+  . /usr/share/openhab2/runtime/bin/inc
+  locateJava >/dev/null
+  VERSION=$("${JAVA}" -version 2>&1 | awk -F '"' '/version/ {print $2}' | sed -e 's/_.*//g; s/^1\.//g; s/\..*//g; s/-.*//g;')
+  if [ "${VERSION}" -lt "8" ]; then
+    echo "### WARNING: Java 8 not detected! You must install Java 8 in order to use openHAB 2.X.X"
```

Comment:
Heyho! How about we make those printouts a bit friendlier?

> ### WARNING: We were not able to detect a Java 8 runtime on your system. It is a prerequisite for openHAB 2 and is needed before openHAB can be executed. Please install a current Java 8 runtime now or check the openHAB documentation for details.

Reply:
Hey @ThomDietrich !

Good idea, I changed the message but didn't think the messages from openHAB stood out much. What do you think of:

![image](https://user-images.githubusercontent.com/8983584/49687813-7d909800-fb00-11e8-91e6-f90e1543df9c.png)

Reply:
I like it very much! I'd remove the `#` though. It could be confused with a root prompt...

Reply:
Fair enough :) Thanks!